src/openepd/api/sync_client.py

```diff
@@ -17,6 +17,7 @@
 
 from requests.auth import AuthBase
 
+from openepd.api.average_dataset.generic_estimate_sync_api import GenericEstimateApi
```

Comment:
```suggestion
from openepd.api.average_dataset.sync_api import GenericEstimateApi
```

Reply:
I don't think this is necessary. 
The end-users would be using the Client as an interface to the library, and making a layer of re-imports here would add not much value. 
Or maybe I am missing a point?

Reply:
Just for consistency with others.